src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1800,65 +1800,47 @@
   <data name="None" xml:space="preserve">
     <value>None</value>
   </data>
-  <data name="HighDpiOption_UseDPIOnLogin" xml:space="preserve">
-    <value>On Windows login</value>
+  <data name="CompatibilityOnWindowsLogon" xml:space="preserve">
+    <value>On Windows log on</value>
   </data>
-  <data name="HighDpiOption_UseDPIOnProgramStart" xml:space="preserve">
-    <value>On program start</value>
+  <data name="CompatibilityOnProgramStart" xml:space="preserve">
+    <value>On this program start</value>
```

Comment:
Isn't "this" implied?

Reply:
Idk. Just followed with his design.
FE's one is not good enough.

![image](https://github.com/files-community/Files/assets/62196528/b2c9049f-4834-4157-9006-b2fc8c5b13bf)

Reply:
Okay